lib/cc/engine/analyzers/violation.rb

```diff
@@ -88,6 +95,16 @@ def read_up_path
           )
         end
 
+        def fingerprint
+          digest = Digest::MD5.new
+          digest << current_sexp.file
+          digest << "-"
```

Comment:
These aren't needed because the whole thing is being hashed and the digest library itself handles the chunks